### DISCUSSION TITLE
Is there a way to have an event not trigger a transition?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [statelyai/xstate#1740] [Q&A]
I have this chart: https://xstate.js.org/viz/?gist=c8caac24b701ce124bc8e7ffbaa43ac7

for context, this is going to control a card game, the "player's hand". `ADJUST` is the event that will be called on window resize, because on the playing cards will need their positions adjusted / scaled / etc

I'm trying to figure out if the `ADJUST` event can leave the current state. 
If you click `SELECT` and then click `ADJUST`, it seems the state is reset. 
Is there a way to prevent that? I'd like to remain in the `selected` state when `ADJUST` happens.

**Comment by davidkpiano** [comment]
Might just be a bug with the older version of XState that is used.

Here, it's visualized in the inspector: https://codesandbox.io/s/focused-jang-fmls9?file=/src/index.js (make sure to enable popups) and it seems like it does the correct behavior.

**Reply by NullVoxPopuli** [reply]
ooo fancy :+1: thanks!
